From 5177936a2bd4aa9e0badea94e48aee1a939e467a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B6ren=20Tempel?= <soeren+git@soeren-tempel.net>
Date: Fri, 17 Apr 2020 12:21:48 +0200
Subject: [PATCH] Revert "abuild: include $pkgname in temporary rootbld
 directory"

This commit broke abuild-rmtemp which has a prefix check to ensure it
doesn't remove non-abuild directory. Revert the commit instead of
adjusting the prefix check for now.

This reverts commit d0828c06557773f252b645618500699a941bd091.
---
 abuild.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/abuild.in b/abuild.in
index 2af9452..fbe2fbd 100644
--- a/abuild.in
+++ b/abuild.in
@@ -2257,7 +2257,7 @@ rootbld() {
 
 	mkusers
 
-	BUILD_ROOT=$(mktemp -d "/var/tmp/abuild-$pkgname.XXXXXXXXXX")
+	BUILD_ROOT=$(mktemp -d /var/tmp/abuild.XXXXXXXXXX)
 
 	mkdir -p "$BUILD_ROOT/proc" "$BUILD_ROOT/etc/apk/keys" \
 		"$BUILD_ROOT/$HOME/.abuild" "$BUILD_ROOT/$aportsgit" \
